### PR TITLE
Calculate twist in hrpsys seq state ros bridge

### PR DESCRIPTION
**Pull request by orikuma**
This PR includes following changes:
- calculate angular velocity as twist.angular
- calculate covariance of pose using constant covariance of velocity

NOTE: Current twist.linear is velocity, not strict linear twist.

### CODE REVIEW

File: hrpsys_ros_bridge/src/HrpsysSeqStateROSBridge.cpp [hunk 1]
```diff
@@ -546,30 +546,86 @@ RTC::ReturnCode_t HrpsysSeqStateROSBridge::onExecute(RTC::UniqueId ec_id)
...
       if (dt > 0) {
-        odom.twist.twist.linear.x = (odom.pose.pose.position.x - prev_odom.pose.pose.position.x) / dt;
-        odom.twist.twist.linear.y = (odom.pose.pose.position.y - prev_odom.pose.pose.position.y) / dt;
-        odom.twist.twist.linear.z = (odom.pose.pose.position.z - prev_odom.pose.pose.position.z) / dt;
-        odom.twist.twist.angular.x = (rpy(0) - prev_rpy(0)) / dt;
-        odom.twist.twist.angular.x = (rpy(1) - prev_rpy(1)) / dt;
-        odom.twist.twist.angular.x = (rpy(2) - prev_rpy(2)) / dt;
-        odom.twist.covariance = odom.pose.covariance;
+        hrp::Matrix33 prev_R = hrp::rotFromRpy(prev_rpy[0], prev_rpy[1], prev_rpy[2]);
+        hrp::Vector3 omega = (R * hrp::omegaFromRot(R * prev_R.transpose())) / dt; // R = exp(omega*dt) * prev_R, omegaFromRot returns matrix_log
```
Comment (snozawa) [inline hrpsys_ros_bridge/src/HrpsysSeqStateROSBridge.cpp]: Is this correct?

How about this?:
```
        hrp::Vector3 omega = (prev_R * hrp::omegaFromRot( prev_R.transpose() * R)) / dt;
```

**Comment by orikuma** [comment]
I think omega is represented as global coordinates in R = exp(omega*dt) * prev_R and as prev_R coordinates in R = prev_R * exp(omega*dt).
Therefore, multipling R to hrp::omegaFromRot(R * prev_R.transpose()) is incorrect.
I fixed the mistake.

**Comment by snozawa** [comment]
Travis fails because of gazebo error
```
]2;[build] 11/12]2;[build] 11/12]2;[build] 11/12[[36mhrpsys_gazebo_tutorials[0m] [36m-- [0mUsing these message generators:[33m gencpp;geneus;genlisp;genpy[0m [0m[0m
]2;[build] 11/12]2;[build] 11/12]2;[build] 11/12]2;[build] 11/12]2;[build] 11/12]2;[build] 11/12]2;[build] 11/12]2;[build] 11/12[[36mhrpsys_gazebo_tutorials[0m] [31m[1mCMake Error[0m at /opt/ros/hydro/share/catkin/cmake/catkinConfig.cmake:75 (find_package):[0m [0m[0m
]2;[build] 11/12[[36mhrpsys_gazebo_tutorials[0m]   Could not find a configuration file for package hrpsys_gazebo_general.[0m [0m[0m
]2;[build] 11/12[[36mhrpsys_gazebo_tutorials[0m] [0m                                                      [0m[0m
]2;[build] 11/12[[36mhrpsys_gazebo_tutorials[0m]   Set hrpsys_gazebo_general_DIR to the directory containing a CMake[0m [0m[0m
]2;[build] 11/12[[36mhrpsys_gazebo_tutorials[0m]   configuration file for hrpsys_gazebo_general.  The file will have one of[0m [0m[0m
]2;[build] 11/12[[36mhrpsys_gazebo_tutorials[0m]   the following names:[0m                                [0m[0m
]2;[build] 11/12[[36mhrpsys_gazebo_tutorials[0m] [0m                                                      [0m[0m
]2;[build] 11/12[[36mhrpsys_gazebo_tutorials[0m]     hrpsys_gazebo_generalConfig.cmake[0m                 [0m[0m
]2;[build] 11/12[[36mhrpsys_gazebo_tutorials[0m]     hrpsys_gazebo_general-config.cmake[0m                [0m[0m
]2;[build] 11/12[[36mhrpsys_gazebo_tutorials[0m] [0m                                                      [0m[0m
]2;[build] 11/12[[36mhrpsys_gazebo_tutorials[0m] [36m[4mCall Stack (most recent call first):[0m[0m                  [0m[0m
]2;[build] 11/12[[36mhrpsys_gazebo_tutorials[0m]   CMakeLists.txt:4 (find_package)[0m                     [0m[0m
]2;[build] 11/12[[36mhrpsys_gazebo_tutorials[0m] [0m                                                      [0m[0m
]2;[build] 11/12[[36mhrpsys_gazebo_tutorials[0m] [0m                                                      [0m[0m
]2;[build] 11/12[[36mhrpsys_gazebo_tutorials[0m] [36m-- [0mchecking for module 'openhrp3.1'[0m                   [0m[0m
]2;[build] 11/12[[36mhrpsys_gazebo_tutorials[0m] [36m-- [0m  found openhrp3.1, version 3.1.8[0m                  [0m[0m
]2;[build] 11/12[[36mhrpsys_gazebo_tutorials[0m] [36m-- [0mchecking for module 'hrpsys-base'[0m                  [0m[0m
]2;[build] 11/12[[36mhrpsys_gazebo_tutorials[0m] [36m-- [0m  found hrpsys-base, version 3.1[0m                   [0m[0m
]2;[build] 11/12[[36mhrpsys_gazebo_tutorials[0m] [36m-- [0mchecking for module 'collada_urdf_jsk_patch'[0m       [0m[0m
]2;[build] 11/12[[36mhrpsys_gazebo_tutorials[0m] [36m-- [0m  package 'collada_urdf_jsk_patch' not found[0m       [0m[0m
]2;[build] 11/12]2;[build] 11/12]2;[build] 11/12]2;[build] 11/12[[36mhrpsys_gazebo_tutorials[0m] [36m-- [0mConfiguring incomplete, errors occurred![0m           [0m[0m
]2;[build] 11/12[[36mhrpsys_gazebo_tutorials[0m] [1m[31m<==[0m '[1m/home/travis/ros/ws_rtmros_common/build/hrpsys_gazebo_tutorials/build_env.sh /usr/bin/cmake /home/travis/ros/ws_rtmros_common/src/rtmros_tutorials/hrpsys_gazebo_tutorials -DCATKIN_DEVEL_PREFIX=/home/travis/ros/ws_rtmros_common/devel -DCMAKE_INSTALL_PREFIX=/home/travis/ros/ws_rtmros_common/install[0m' [31mfailed with return code[0m '[1m1[0m'[0m [0m[0m
]2;[build] 11/12[1m[31mFailed[0m   [32m<==[0m [36mhrpsys_gazebo_tutorials    [0m [ [33m2.7 seconds[0m ][0m                        [0m[0m
[build] Calculating new jobs...                                                 [0m[0m
]2;[build] 11/12]2;[build] 11/12]2;[build] 11/12]2;[build] There were '1' [1m[31merrors[0m:[0m                                                  [0m[0m

[31mFailed[0m to build package '[36mhrpsys_gazebo_tutorials[0m' because the following command
```
https://github.com/start-jsk/rtmros_common/pull/794
fails because of the same error.

In the travis log, install from deb fails:
```
[1mexecuting command [sudo -H apt-get install -y -qq ros-hydro-hrpsys-gazebo-general][0m
E: Unable to locate package ros-hydro-hrpsys-gazebo-general
```

Does anyone know the reason??

**Comment by garaemon** [comment]
I think it's a problem of timing of releasing shadow-fixed packages.

**Comment by snozawa** [comment]
I see. Thanks.

**Pull request merged**